.github/workflows/build.yml

```
@@ -36,16 +36,25 @@ jobs:
       - name: ccache
         uses: hendrikmuhs/ccache-action@5ebbd400eff9e74630f759d94ddd7b6c26299639 # v1.2
+        if: ${{ matrix.platform != 'windows-latest' }}
         with:
           key: ${{ github.job }}-${{ matrix.platform }}
           verbose: 1
-          create-symlink: ${{ matrix.platform != 'windows-latest' }}
+          create-symlink: true
+
+      - name: Setup cache for clcache
+        if: ${{ matrix.platform == 'windows-latest' }}
+        uses: actions/cache@v3
```

actionlint (1.7.8)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)